src/components/AnswersTable.tsx

```diff
@@ -63,7 +63,10 @@ export const AnswersTable = (props: AnswersTableProps) => {
 
   const answersHtml = answers.map((answer, i) => {
     const additionalProps: { answer: AnswerType, iter: number, key: string }
-      = { answer, iter: i, key: `${questionAnswerProps.qid}-option-${i}` };
+      = { answer: {...answer, question_id: parseInt(question.id.toString(), 10) },
```

Comment:
This could be a string from the Tutor side, or a number from assessments... this doesn't feel like the best pattern, but this satisfies the type for either render case.

Reply:
Yea I would change it to do a type check and only parseint on string, but does tutor not care about it being a number after that?

Reply:
Tutor won't use this field, but shouldn't cause any issues being in the object. I could also look at only adding it when questionsStates (the assessment render case) is present 🤔